package.json

```diff
@@ -12,7 +12,7 @@
     "test": "vitest run",
     "test:watch": "vitest watch",
     "coverage": "vitest run --coverage",
-    "storybook": "NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6006",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Oops, I think this was a merge conflict that got resolved incorrectly-- can you add this back?

Reply:
I put it back in, but I get the following error when trying to run the storybook script on my branch:
![Screenshot 2023-01-30 160334](https://user-images.githubusercontent.com/108034667/215624568-408918b0-968c-4af5-a7f4-d94f34dd68ab.png)